packages/story-editor/src/components/library/panes/media/common/mediaElement.js

```diff
@@ -261,11 +261,11 @@ Element.propTypes = {
  * @return {null|*} Element or null if does not map to video/image.
  */
 function MediaElement(props) {
-  const { isTranscoding } = props.resource;
+  const { isTranscoding, isMuting, isTrimming } = props.resource;
```

Comment:
It will be super simple to tweak this in https://github.com/google/web-stories-wp/issues/9577